### ISSUE TITLE
[Accessibility] "Show password" checkbox doesn't provide label or accessible name

### ISSUE THREAD

**Issue by javido** [nextcloud/server#15965]
### Steps to reproduce
1. Open admin account creation form
2. With screen reader (NVDA  or Orca) press tab key until give focus to the checkbox

### Expected behaviour
Tell us what should happen
Screen reader should announce "Show password" unchecked chebox
### Actual behaviour
Tell us what happens instead
Screen reader only announces that there is a checkbox, but users don't know this is for showing the password.
### Server configuration

**Operating system**:
Ubuntu 19.04, Mozilla Firefox and Orca screen reader. 
**Web server:**
Apache 2
**Database:**
MariaDB
**PHP version:**
PHP 7.2
**Nextcloud version:** (see Nextcloud admin page)
Nextcloud Master
**Updated from an older Nextcloud/ownCloud or fresh install:**
Fresh installation 
**Where did you install Nextcloud from:**
Git repository

**Comment by kesselb** [comment]
You are referring to the setup page right?

https://github.com/nextcloud/server/blob/7fea53184783bb7438f46169d239b635fa21319a/core/templates/installation.php#L47-L53

How to fix this? @jancborchardt

**Comment by javido** [comment]
Yes, sure. Problems seems that label tag it's empty. label for show

**Comment by javido** [comment]
Label for show field should have "show password" text. If you don't want to make this text visible aria-label on input type checkbox it's ok.
BTW How I can insert html code in my comments? I am not familiarized with Github yet.

**Comment by kesselb** [comment]
> BTW How I can insert html code in my comments? I am not familiarized with Github yet.

Like this https://help.github.com/en/articles/creating-and-highlighting-code-blocks

**Comment by Sibin-Thomas** [comment]
Hey Can I fix this issue?

**Comment by kesselb** [comment]
> Hey Can I fix this issue?

We appreciate your help :+1: Please note that there is already a pull request https://github.com/nextcloud/server/pull/16642 with a proposed solution. I'm not sure what the status is.
